### DISCUSSION TITLE
My Github Pages is not opening.

### DISCUSSION THREAD

**Discussion by niranjank2022** [community/community#60898] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I attempted to add a Custom domain to my Github pages website https://niranjank2022.github.io/Cluster-of-Projects/. But, I entered the n-apps.gen in the Custom domain box. Then, GitHub showed me that it is not possible to have that domain saying some DNS error. So, I removed that domain. But, now I am not able to open my site  via this link https://niranjank2022.github.io/Cluster-of-Projects/

Please help me how to solve this problem.

**Comment by enginestein** [comment]
First of all you can only add those custom domains which are owned by you and I don't think that you are trying to enter a domain that's bought by you and even if I am wrong, it's clear that you are inputting a subdomain. To make a custom domain work you need to go to the panel, the admin panel of the domain. The panel is available on the website where you bought the domain. For example, GoDaddy has it's own admin panel only if you buy a domain, all domain distributing websites have that. Now, if you want to add a custom domain you need to go to the DNS settings of your domain in admin panel and github pages will provide you IP addresses which are supposed to be value of a DNS address. It's not that hard if you look around the admin panel. However, I am providing you an image of what an admin panel looks like

About not being able to open the site on normal github pages domain, it takes a bit of time to reload, I clicked the URL you provided and the website works, it just takes a bit time to reload on github pages domain.

**Reply by niranjank2022** [reply]
I didn't buy any subdomains. I didn't know about that and I randomly entered this domain www.n-apps.com. Whenever I click the normal GitHub pages link, it is opening in that link only. How, can I resolve this?

**Reply by CypherpunkSamurai** [reply]
Remove it from Repository Settings -> Pages -> Domains. Remove the one that is not yours, you'll get your `github.io` domain.

**Comment by enginestein** [comment, marked as answer]
Subdomains are not bought, if you have your own domain you can create
subdomains from admin panel. Apart from that, it is actually not opening
the n-apps.com, it is trying to open it but as it is not your domain and is
not verified from DNS addresses, the page does not load. This might look a
confusing error but should not be an big issue. You can either go to custom
domains section and remove www.n-apps.com domain from that place ( the
place where you typed this domain name ) or you can remove the repository,
create a new repository and start deploying it.

Cloud hosting is not a tough task, once you understand it you get it all.
Domains are not much expensive too however some of them are. Mistakes may
happen but next time, work carefully, these things are quite dangerous and
may not work as you want it to work. Once you understand github pages, you
will understand other hosting services easily too. If you need any further
assistance, let me know.

**Reply by niranjank2022** [reply]
@enginestein Thanks for your kind and quick responses. This is what it shows in Custom Domain section! Is there any other way to remove the custom domain? If not and if I have to remove the repo, I can't create a repo with the same name and deploy, right?

![Screenshot (40)](https://github.com/community/community/assets/112152164/09a6d544-0fde-4f23-bc4e-f751f3822bad)

**Reply by CypherpunkSamurai** [reply]
When you remove a repo (permanently wipe it off your account) it will reset GitHub Domain configs. You can then recreate that repo to claim that GitHub.io domain again.

Follow Removing steps:
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#removing-a-custom-domain

**Reply by CypherpunkSamurai** [reply]
Also please flush your DNS and clean you browser cache and try after a clean boot.

**Reply by enginestein** [reply]
There is no problem with the hosted project. It is working as intended so no need to do anything like that.

**Comment by enginestein** [comment]
@niranjank2022 Well, there is actually no problem with your hosting. I just checked, go to
https://niranjank2022.github.io/Cluster-of-Projects/ and you will find your
website. No need to reset the project or create a new repo, the custom
domain is removed, your website is working well on github pages domain
which is https://niranjank2022.github.io/Cluster-of-Projects/

If there is still any problem regarding github pages hosting or any other
hosting, feel free to let me know.

**Reply by niranjank2022** [reply]
Thanks @enginestein and @CypherpunkSamurai for your responses. I figured out that the page was opening. But, it still opened n-apps.com in my Chrome browser. So, I cleared the cache as @CypherpunkSamurai mentioned. Onca again thanks for both of you.
